pkg/replication/README.md

```diff
@@ -1,44 +0,0 @@
-# Replication: facilities for scheduling a pod across multiple nodes
```

Comment:
whoops, I must have blown the readme away when i was replacing the old package with the rewritten one. Will restore